qiskit_experiments/library/calibration/fine_drag.py

```diff
@@ -0,0 +1,265 @@
+        options.amp = 1.0
+
+        return options
+
+    def __init__(self, qubit: int):
+        """Setup a fine amplitude experiment on the given qubit.
+
+        Args:
+            qubit: The qubit on which to run the fine amplitude calibration experiment.
+        """
+        super().__init__([qubit])
+
+    @staticmethod
+    def _pre_circuit() -> QuantumCircuit:
+        """Return the quantum circuit to apply before repeating the Rp - Rz - Rp - Rz gates."""
+        return QuantumCircuit(1)
+
+    @staticmethod
+    def _post_circuit() -> QuantumCircuit:
+        """Return the quantum circuit to apply after repeating the Rp - Rz - Rp - Rz gates."""
+        return QuantumCircuit(1)
+
+    def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
+        """Create the circuits for the fine DRAG calibration experiment.
+
+        Args:
+            backend: A backend object.
+
+        Returns:
+            A list of circuits with a variable number of gates. Each gate has the same
+            pulse schedule.
+        """
+        schedule, circuits = self.experiment_options.schedule, []
+
+        drag_gate = Gate(name=schedule.name, num_qubits=1, params=[])
+
+        for repetition in self.experiment_options.repetitions:
+            circuit = self._pre_circuit()
+
+            for _ in range(repetition):
+                circuit.append(drag_gate, (0,))
+                circuit.rz(np.pi, 0)
+                circuit.append(drag_gate, (0,))
+                circuit.rz(np.pi, 0)
+
+            circuit.ry(np.pi / 2, 0)  # Maps unwanted Z rotations to qubit population.
+
+            circuit.compose(self._post_circuit(), inplace=True)
+
+            circuit.measure_all()
+            circuit.add_calibration(schedule.name, self._physical_qubits, schedule, params=[])
+
+            circuit.metadata = {
+                "experiment_type": self._type,
+                "qubits": self.physical_qubits,
+                "xval": repetition,
+                "unit": "gate number",
+            }
+
+            circuits.append(circuit)
+
+        return circuits
+
+
+class FineXDrag(FineDrag):
+    """Class to fine calibrate the DRAG parameter of an X gate.
+
+    # section: see_also
+        qiskit_experiments.library.calibration.fine_drag.FineDrag
+    """
+
+    @staticmethod
+    def _pre_circuit() -> QuantumCircuit:
+        """Return the quantum circuit done before the Rp - Rz - Rp - Rz gates."""
+        return QuantumCircuit(1)
+
+
+class FineSXDrag(FineDrag):
+    """Class to fine calibrate the DRAG parameter of an SX gate.
+
+    # section: see_also
+        qiskit_experiments.library.calibration.fine_drag.FineDrag
+    """
+
+    @staticmethod
+    def _pre_circuit() -> QuantumCircuit:
+        """Return the quantum circuit with an sx gate before the Rp - Rz - Rp - Rz gates."""
+        circ = QuantumCircuit(1)
+        circ.sx(0)
+        return circ
+
+
+class FineXDragEF(FineDrag):
```

Comment:
I haven't worked much with qutrits. Does this work? Does the final ry rotation need to be an EF ry?

Reply:
I've used DRAG cal for qutrit and seems like it finds reasonable values (but in different approach). However I've never tested the effect of it since I didn't have time to implement qutrit RB.

Reply:
Good point about the final rotation. I ended up removing this in 9d89389 perhaps it might be better to have a PR dedicated to cals in the EF subspace with a demo NB to go along.